### DISCUSSION TITLE
APISIX Standalone - Warning message for maximum number of open file descriptors

### DISCUSSION THREAD

**Discussion by nphariharan** [apache/apisix#10098] [General]
I am using APISIX 3.4 version(Standalone mode). How to address/resolve the below warning?

/usr/local/openresty//luajit/bin/luajit ./apisix/cli/apisix.lua init
Warning! Current maximum number of open file descriptors [1024] is not greater than 1024, please increase user limits by execute 'ulimit -n <new user limits>' , otherwise the performance is low.
/usr/local/openresty//luajit/bin/luajit ./apisix/cli/apisix.lua init_etcd
2023/08/14 12:36:53 [alert] 218#218: setrlimit(RLIMIT_NOFILE, 20480) failed (1: Operation not permitted)
2023/08/14 12:36:53 [alert] 220#220: setrlimit(RLIMIT_NOFILE, 20480) failed (1: Operation not permitted)
2023/08/14 12:36:53 [alert] 221#221: setrlimit(RLIMIT_NOFILE, 20480) failed (1: Operation not permitted)
2023/08/14 12:36:53 [alert] 222#222: setrlimit(RLIMIT_NOFILE, 20480) failed (1: Operation not permitted)
2023/08/14 12:36:53 [alert] 219#219: setrlimit(RLIMIT_NOFILE, 20480) failed (1: Operation not permitted)

**Comment by kayx23** [comment]
> please increase user limits by execute 'ulimit -n ' , otherwise the performance is low.

**Comment by flylan** [comment]
You should be using the Docker version of Apisix, right? As far as I know, the Docker version of the Apisix image uses the Apisix user to run the Apisix service, not the root user. If root permission commands need to be executed during runtime, there will be problems
